src/Sarif.Viewer.VisualStudio/SarifSnapshot.cs

```diff
@@ -0,0 +1,289 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Windows.Forms;
+
+using EnvDTE;
+
+using Microsoft.CodeAnalysis.Sarif.Driver;
+using Microsoft.CodeAnalysis.Sarif.Driver.Sdk;
+using Microsoft.CodeAnalysis.Sarif.Sdk;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Shell.TableManager;
+using Microsoft.VisualStudio.TextManager.Interop;
+
+namespace SarifViewer
+{
+    class SarifSnapshot : TableEntriesSnapshotBase
+    {
+        private string _projectName;
```

Comment:
I see that this class uses `_` prefix on locals (consistent with Roslyn convention) but classes above do not (which is why you had to write `this.package = package`). Would you consider moving to `_` uniformly?